### PR TITLE
Rename files: related to tPAC analysis

### PR DESCRIPTION
**Pull request by SoheilaSamiee**

### CODE REVIEW

File: toolbox/process/functions/process_pac_comod.m [hunk 1]
```diff
@@ -1,4 +1,4 @@
-function varargout = process_pac_semiComod( varargin )
+function varargout = process_pac_comod( varargin )
 % process_pac_semiComod: Extract semi-comodulogram from tPAC maps.
```
Comment (martcous) [inline toolbox/process/functions/process_pac_comod.m]: Function name in description still incorrect.

Comment (martcous) [reply toolbox/process/functions/process_pac_comod.m]: Actually, 'process_pac_semiComod' is still present many times in the file. Make sure you Search & replace all of the instances.

File: toolbox/process/functions/process_pac_fp_map.m [hunk 2]
```diff
@@ -1,4 +1,4 @@
-function varargout = process_pac_fpMap( varargin )
+function varargout = process_pac_fp_map( varargin )
 % process_pac_fpMap: Extract frequency for phase map from tPAC matrices.
```
Comment (martcous) [inline toolbox/process/functions/process_pac_fp_map.m]: Same comment here, 'process_pac_fpMap' is still present many times in the file.

**Comment by SoheilaSamiee** [comment]
Thank you for the note,
I will do so,

**Pull request merged**